get-started/contributing/index.md

```diff
@@ -1,26 +1,61 @@
 ---
 layout: custom
-title: Contributing to Monero
+title: Improving Monero
 ---
 <div markdown="1" class="text-center container description">
-Monero is an open source, community driven project. We strive to be a welcoming community. Perhaps you want to contribute?
+Monero is an open-source, community-driven project. Described below are several ways to support the project.
 </div>
 
 <div class="contribute">
    
     <section class="container">
-          <div class="row">                
+          <div class="row">         
+               <!-- full block-->
+               <div class="full col-lg-12 col-md-12 col-sm-12 col-xs-12">
+                        <div class="info-block text-adapt">
+                            <div class="row center-xs">
+                                <div class="col"><h2>Support the Network</h2></div>
+                            </div>
+<div class="row start-xs" markdown="1">
+
+### Code
+Monero is primarily written in C++. As it is a decentralized project, no permission is required to generate new code or modify existing code. Pull requests are merged based on community consensus. See the <a href="https://github.com/monero-project"> repositories</a> and outstanding <a href="https://github.com/monero-project/monero/issues">issues.</a>
```

Comment:
Possibly change wording here too:
```
Monero is primarily written in C++. Because it is a decentralized and open source project,  anyone is welcome to add or make changes to existing code. Pull requests are merged based on community consensus. See the <a href="https://github.com/monero-project"> repositories</a> and outstanding <a href="https://github.com/monero-project/monero/issues">issues.</a>
```

Reply:
Don't have to use mine, but I think yours could flow a bit better.

Reply:
@QuickBASIC IMO your version does sound much more natural